swig/CMakeLists.txt

```diff
@@ -53,17 +55,22 @@ add_jar(
     # "${CMAKE_SWIG_OUTDIR}/ParameterInitSaxe.java" (I AM NOT ACTUALLY IMPLEMENTED IN DYNET)
     "${CMAKE_SWIG_OUTDIR}/ParameterInitFromFile.java"
     "${CMAKE_SWIG_OUTDIR}/ParameterInitFromVector.java"
+    "${CMAKE_SWIG_OUTDIR}/ParameterStorage.java"
+    "${CMAKE_SWIG_OUTDIR}/ParameterStorageBase.java"
+    "${CMAKE_SWIG_OUTDIR}/ParameterStorageVector.java"
     "${CMAKE_SWIG_OUTDIR}/RmsPropTrainer.java"
     "${CMAKE_SWIG_OUTDIR}/RNNBuilder.java"
     "${CMAKE_SWIG_OUTDIR}/SimpleRNNBuilder.java"
     "${CMAKE_SWIG_OUTDIR}/SimpleSGDTrainer.java"
     "${CMAKE_SWIG_OUTDIR}/StringVector.java"
     "${CMAKE_SWIG_OUTDIR}/SWIGTYPE_p_dynet__LookupParameterStorage.java"
+    "${CMAKE_SWIG_OUTDIR}/SWIGTYPE_p_dynet__ParameterStorage.java"
```

Comment:
Can you double check that you need the three dynet__*Storage*.java pointer classes now that they also have their own classes. E.g., delete all the java files and rebuild to see if they're still there.